mappings/net/minecraft/util/registry/RegistryEntryListCodec.mapping

```diff
@@ -1,9 +1,44 @@
 CLASS net/minecraft/class_6898 net/minecraft/util/registry/RegistryEntryListCodec
 	FIELD field_36480 registry Lnet/minecraft/class_5321;
 	FIELD field_36481 entryCodec Lcom/mojang/serialization/Codec;
-	FIELD field_36482 entryListCodec Lcom/mojang/serialization/Codec;
+	FIELD field_36482 directEntryListCodec Lcom/mojang/serialization/Codec;
+	FIELD field_36483 entryListStorageCodec Lcom/mojang/serialization/Codec;
 	METHOD <init> (Lnet/minecraft/class_5321;Lcom/mojang/serialization/Codec;Z)V
 		ARG 1 registry
 		ARG 2 entryCodec
-	METHOD method_40382 (Lcom/mojang/serialization/Codec;Z)Lcom/mojang/serialization/Codec;
+		ARG 3 alwaysSerializeAsList
+	METHOD decode (Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 ops
+		ARG 2 input
+	METHOD encode (Ljava/lang/Object;Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 entryList
+		ARG 2 ops
+		ARG 3 prefix
+	METHOD method_40380 (Lcom/mojang/datafixers/util/Either;)Ljava/util/List;
+		ARG 0 either
+	METHOD method_40381 (Lcom/mojang/datafixers/util/Pair;)Lcom/mojang/serialization/DataResult;
+		ARG 0 pair
+	METHOD method_40382 createDirectEntryListCodec (Lcom/mojang/serialization/Codec;Z)Lcom/mojang/serialization/Codec;
 		ARG 0 entryCodec
+		ARG 1 alwaysSerializeAsList
+			COMMENT whether to always serialize the list as a list
+			COMMENT instead of serializing as one entry if the length is {@code 0}
```

Comment:
Could this comment be added to other similar `alwaysSerializeAsList` parameters?